src/measure_theory/integral/integrable_on.lean

```diff
@@ -217,6 +217,10 @@ lemma integrable.indicator (h : integrable f μ) (hs : measurable_set s) :
   integrable (indicator s f) μ :=
 h.integrable_on.indicator hs
 
+lemma integrable_on.indicator' (h : integrable_on f s μ) (ht : measurable_set t) :
```

Comment:
I would rename the old `integrable_on.indicator` to `integrable_on.integrable_indicator` (I know, it's longer, so feel free to ignore).

Reply:
Yes, I agree that using `null_measurable` more would be better. But this will have to wait for another PR :-)